### ISSUE TITLE
Lack of documentation

### ISSUE THREAD

**Issue by jweger1988** [phac-nml/quasitools#2]
Hello, thanks for developing this tool. It looks pretty awesome and I would love to try it out. However, the documentation is not very clear or not present. Is it possible to see a little more documentation on how to run quasitools complexity for example? Thanks in advance for your help.

**Comment by emarinier** [comment]
Hello,

Sorry about the lack of documentation. It's something that we're starting to work on now.

The complexity tool reports quasispecies complexity measures recommended by [Gregori et al. 2016](https://www.sciencedirect.com/science/article/pii/S004268221630037X).

Currently, the tool takes a single, aligned, FASTA file as input and reports the measures of complexity to the output. After quasitools is installed, you can run the complexity command as follows:

`quasitools complexity aligned.fasta`

**Comment by jweger1988** [comment]
Hey, 

Thanks for the quick reply. So the input would be the output of some haplotype calling program then? It doesn't appear quasitools has a haplotype calling function, or am I missing it?  Thanks again.

**Comment by emarinier** [comment]
No, I don't believe there's any other quasitools tool designed to work directly with the complexity tool.

One situation where this tool might be useful is if you have sequencing data of a particular region, perhaps a portion of the Pol gene in HIV, and you want to understand the quasispecies complexity of that region. You might align the reads to a reference (using Bowtie2, or similar) and organize them into an (aligned) FASTA file and use that as input to complexity.
